examples/dune

```
@@ -8,6 +8,18 @@
   (:standard -w -69))
  (package ortac-examples))
 
+(rule
+ (alias runtest)
+ (mode promote)
+ (targets dune.lwt_dllist.inc)
+ (action
+  (with-stdout-to
+   %{targets}
```

```suggestion
 (action
  (with-stdout-to
   dune.lwt_dllist.inc
```
The target file `with-stdout-to` writes to is automatically added to the targets, and I think the shorter version is also nicer to read.
If you adopt this suggestion, there are a couple of other occurrences in the same file and a few others elsewhere in the repository (the ones in generated rules are less interesting to change, as they are not models of rules users should write by hand).